### PR TITLE
Added information on polls, fixed error on C++ with unit test, and moved active code items together.

### PR DESCRIPTION
**Pull request by barbarer**

### CODE REVIEW

File: old-pavement.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import paver
```
Comment (bnmnetp) [inline old-pavement.py:1]: please remove this from the PR

File: pavement.py [hunk 2]
```diff
@@ -1,57 +1,87 @@
...
+home_dir = os.getcwd()
 
-######## CHANGE THIS ##########
 project_name = "overview"
-###############################
 
-master_url = "https://runestone.academy"
+#master_url = None
+#if master_url is None:
+#    if 'RSHOST' in os.environ:
```
Comment (bnmnetp) [inline pavement.py:23]: We use `black` as the master formatter for all our .py files now.  I don't see any substantive changes, so please take this out of the PR.

**Comment by barbarer** [comment]
I changed it to only update overview.rst

**Comment by bnmnetp** [comment]
Are you going to open a new PR with the changes to overview?

**Comment by barbarer** [comment]
I am trying to figure out how to remote the files that don't need to be in
the PR.  How do I do that?

Thank You,

Dr. Barbara Ericson
Assistant Professor, School of Information
University of Michigan

**Comment by bnmnetp** [comment]
Since it was all on the master branch of your fork, I think I would copy overview.rst to a safe place and make a new fork.  Then apply your changes to overview to the new fork.  Otherwise you are going to have to do some fancy stuff with git reset to undo the commits you made that added the .DS_Store file and old-pavement and the changes you made to pavement.

I try to do all of my work that I"m going to PR on a branch as that allows me an easy escape.  If you had made these changes on a branch you could just make a new branch and then use git checkout otherbranch overview.rst to get only the changes to overview transferred over to the new branch.  Then simple git branch -D oldbranch to get rid of it.

**Pull request closed without merging**